QUARKUS-6752.md

```diff
@@ -0,0 +1,50 @@
+# QUARKUS-6752 Testing and Verification of Semeru CE 21 & 17 on x86_64 (RHEL 8)
+
+JIRA: https://issues.redhat.com/browse/QUARKUS-6752
+
+Upstream issue: https://github.com/quarkusio/quarkus/issues/50570
+
+Semeru JDK is an IBM's release of JDK, using OpenJDK's libraries and IBM's OpenJ9 JVM [[1]](https://developer.ibm.com/languages/semeru-runtimes/).
```

Comment:
OpenEdition is the same binary as CertifiedEdition. I think this was discussed in the Semeru Slack channel (or maybe email thread, I will try to find it when near the work ntb)

https://github.com/ibmruntimes/semeru21-binaries/releases
https://github.com/ibmruntimes/semeru17-binaries/releases